cves/CVE-2013-0847.yml

```diff
@@ -326,8 +323,8 @@ lessons:
     applies:
     note:
   complex_inputs:
-    applies:
-    note:
+    applies: true
+    note: 'Exploiting this vulnerability requires a crafted long ID3 metadata container'
```

Comment:
Is it `IP3` or `ID3`? I see both throughout your CVE.

Reply:
It's ID3. Thanks for catching that!